app/assets/javascripts/admin/views/editions/_form.js

```diff
@@ -265,6 +266,27 @@
         $(this).parent().hide().next().removeClass('if-js-hide');
         e.preventDefault();
       });
+    },
+
+    showImageUploaderIfCustomImage: function showImageUploaderIfCustomImage() {
+      var $imageUploader = $('.image_section .js-show-image-uploader');
+      var $customImage = $('#edition_image_display_option_custom_image');
+      var $radioButtons = $('.image_section input[type=radio]');
+
+      function showOrHideImageUploader() {
+        if ($customImage.prop('checked')){
+          $imageUploader.show();
+        } else {
+          $imageUploader.hide();
+        }
+      }
+      $radioButtons.on('change', showOrHideImageUploader);
+      showOrHideImageUploader();
+
+      $('.image_fields a').on('click', function(e) {
+        $(this).parent().hide().next().removeClass('if-js-hide');
```

Comment:
This anonymous function isn't particularly self-documenting, unlike the rest of the (excellent!) JS code above it. Could you use some well named functions/variables to describe what's going on here?

Reply:
Just looked into this and I think the `if-js-hide` class has been removed so I think I can remove the method. It was meant to be for some text that says only to fill in the field if you wanted to add an image but since it isn't marked as required and users can leave the field blank on the form typically we've taken the text out. I'll remove it now.

Sorry I missed it! 🙈